waku/v2/node/wakunode2.nim

```diff
@@ -1025,8 +1027,14 @@ when isMainModule:
     ## Resume history, connect to static nodes and start
     ## keep-alive, if configured.
 
+    #if websocket or secure websocket enabled,
+    #start websocket server with websocketConn as true.
+    if(conf.websocketSupport):
+      waitFor node.start(websocketConn = true)
```

Comment:
I feel like this should be part of the node object at config time probably

Reply:
makes sense